.env.spec

```diff
@@ -2,6 +2,7 @@ CORS_ORIGIN=*
 CORS_METHOD=*
 MARKETPLACE_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/marketplace
 COLLECTIONS_SUBGRAPH_URL=https://api.thegraph.com/subgraphs/name/decentraland/collections-matic-mainnet
+SIGNATURES_SERVER_URL=http://localhost:3000
```

Comment:
We can add whatever we want here as it's only for the specs, so I added a localhost site, but it could be anything we want.

Reply:
yea but once the server is live it should be something like `https://signatures-api.decentraland.org` right? Otherwise you need to have your own local instance running